### DISCUSSION TITLE
v3.7.0-R4.0

### DISCUSSION THREAD

**Discussion by RyanMetcalfeInt8** [intel/openvino-plugins-ai-audacity#318] [Announcements]
# Compatibility :wrench:
:warning: This plugin release is *only* compatible with [Audacity 3.7.0](https://github.com/audacity/audacity/releases/tag/Audacity-3.7.0) 64-bit Release for Windows -- so make sure you have this version of Audacity installed! 

## Changes since previous release:
* Audacity compatibility updated from 3.6.4 to 3.7.0
* Fixed issue that was causing problems when Windows username has special characters (#313, #293) -- Thank you @Courteous-Cheetah for helping to validate!

## :warning: If you've previously installed any of our 3.6.X or 3.5.X plugin releases, the recommendation for upgrade is to *not* uninstall the previous version. Instead, just download this version and select *Install no models* from the drop down list. This way, you won't need to re-download all of the models again.

:warning: **Make sure to select an Audacity 3.7.0 location during installation. If you have multiple Audacity versions installed, the installer may select the wrong location by default -- so be mindful of this, and manually adjust this using the *browse* button if needed**. 

:penguin: Although we are only providing an installer for Windows right now, it can be compiled for Linux using [this build procedure](https://github.com/intel/openvino-plugins-ai-audacity/blob/b6290c294040a23081d0ece90ede86c6f0066613/doc/build_doc/linux/README.md).

# Supported Hardware
As this release is built on top of OpenVINO™ 2024.3.0, it will generally support hardware defined in this list: https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html 

# Need help? :raising_hand:
If you encounter any issues during install procedure, reach out to us for help by filing an issue [here](https://github.com/intel/openvino-plugins-ai-audacity/issues). We are happy to help!

<hr /><em>This discussion was created from the release <a href='https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.7.0-R4.0'>v3.7.0-R4.0</a>.</em>

**Comment by lachinhan** [comment]
I hope you guys will develop it on Reaper Daw

**Comment by Brodvd** [comment]
Hey sorry but the discussions in this repository couldn't be create now? I didn't find the buttom "new discussion"
Update: If I use the app GitHub it work .

![image](https://github.com/user-attachments/assets/a91d08cb-842b-4ee6-90c8-f914b8977602)

**Reply by RyanMetcalfeInt8** [reply]
hmm, they should be enabled. I certainly didn't do anything to disable them. I at least see 'New Discussion' button... but of course I'm admin so it doesn't mean much. Are you logged in?

**Reply by Brodvd** [reply]
Yes, I'm log in, but It Is strange, if I go in an other repository I see the buttom "new discussion", in this repository the buttom there's only in the GitHub android app.
Anyway not Is a problem for me.

**Reply by RyanMetcalfeInt8** [reply]
hmm, interesting, thanks for the heads up. You mean 'New Discussion' button doesn't show up [here](https://github.com/intel/openvino-plugins-ai-audacity/discussions) for you?

**Reply by Brodvd** [reply]
Ehm... there's the problem also in the app GitHub

![IMG_20241202_165402](https://github.com/user-attachments/assets/80592da9-b9af-4115-9cec-591156225a19)

It's a problem in my settings?

**Reply by RyanMetcalfeInt8** [reply]
I'm not sure... I don't even see any admin options around discussion creation permissions, etc.

**Reply by RyanMetcalfeInt8** [reply]
You can always create an 'issue' and I'll promote to a discussion if it's better suited for that.

**Comment by LocutorBorges** [comment]
Por favor, ajudem-me.
Eu instalo o Audacity 3.7 e o OpenVino porém depois de ativar o Modulo o Audacity não abre novamente e aparece apenas um erro com os seguintes dados:

Operating system: Windows NT 10.0.19045 5131 CPU: amd64 family 6 model 30 stepping 5 8 CPUs GPU: UNKNOWN Crash reason: EXCEPTION_ILLEGAL_INSTRUCTION Crash address: 0x1c1553141ea Process uptime: 4 seconds...

E mais um monte de numeros.

Já troquei as Dlls já instalei versões mais antigas, mas não deu certo

**Reply by RyanMetcalfeInt8** [reply]
Hello, try deleting `openvino_intel_npu_plugin.dll` from `C:\Program Files\Audacity`.

**Comment by TriQyup** [comment]
Just waiting for **3.7.1** Release :D

**Reply by RyanMetcalfeInt8** [reply]
Ah yep, just saw that it was released. I'll try to get a new release posted today.

**Reply by RyanMetcalfeInt8** [reply]
okay, it's posted https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.7.1-R4.1

Enjoy! 🤘
